bin/main.py

```diff
@@ -5,14 +5,18 @@
 import sys
 import time
 
+import rx
+from rx import Observable, Observer
+
 from PyQt5.QtCore import QThread, pyqtSignal
 from PyQt5.QtWidgets import QApplication
 from PyQt5.QtWidgets import QMainWindow
 from slackclient import SlackClient
-
+from app.application import Application
 from app.gui.main_window import Ui_MainWindow
 
-token = os.environ['SLACK_API_TOKEN']
+# token = os.environ['SLACK_API_TOKEN']
+token = 'xoxp-132464150004-133138935623-133129000214-c8b0e2d9389ddbbef75d1a29ec099a9d'
```

Comment:
![you-had-one-job34-580x425](https://cloud.githubusercontent.com/assets/1143994/22348676/f853e4b2-e44f-11e6-8467-bff06de38f30.jpg)